app/Models/Extensions/SizeVariants.php

```diff
@@ -178,10 +178,11 @@ public function getThumb(): ?SizeVariant
 	 *                                size variant shall point to
 	 * @param int    $width           the width of the size variant
 	 * @param int    $height          the height of the size variant
+	 * @param int    $filesize        the filesize of the size variant
 	 *
 	 * @return SizeVariant The newly created and persisted size variant
 	 */
-	public function create(int $sizeVariantType, string $shortPath, int $width, int $height): SizeVariant
+	public function create(int $sizeVariantType, string $shortPath, int $width, int $height, int $filesize = -1): SizeVariant
```

Comment:
I know that @ildyria asked you to add a default parameter. However, I haven't yet found the the place where we need to create a size variant without the correct, final size right away. I probably have to take a second review round.

Reply:
OK, found the cause here: https://github.com/LycheeOrg/Lychee/pull/1239#discussion_r820224978

However, as there is only a single place where we need to create a size variant with a dummy file size of `-1` first and even this specific line of code should better not exist (see linked comment), I would rather prefer not to add a default value here. This is only an invitation to create size variants without a proper file size. Optimally, size variants a are only created with a proper file size the same way we also require a proper with and height.

@ildyria What do you think? Can we revert this change?

Reply:
Sure. No strong opinion.

Reply:
Indeed I agree that using a default value only for that case, that will probably change, is not useful. See d02ea7e68.